api/tests/routes/native/v1/account_test.py

```diff
@@ -642,6 +642,63 @@ def test_update_user_profile_reset_recredit_amount_to_show(self, client, app):
         assert user.recreditAmountToShow is None
 
 
+class ConfirmUpdateUserEmailTest:
+    def _initialize_token(self, user, app, new_email):
+        expiration_date = datetime.utcnow() + users_constants.EMAIL_CHANGE_TOKEN_LIFE_TIME
+        token = encode_jwt_payload({"current_email": user.email, "new_email": new_email}, expiration_date)
+        app.redis_client.set(email_update.get_confirmation_token_key(user), token)  # type: ignore [attr-defined]
+        app.redis_client.expireat(email_update.get_confirmation_token_key(user), expiration_date)  # type: ignore [attr-defined]
+        return token
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏mypy ignor les fichiers de test dans la CI
Pas besoin des `# type: ignore` ici